packages/app/src/App.tsx

```diff
@@ -40,6 +40,10 @@ import {
   CatalogIndexPage,
   catalogPlugin,
 } from '@backstage/plugin-catalog';
+
+/* Uncomment this import to enable a customized version of a catalog */
+// import '@internal/plugin-catalog-customized';
```

Comment:
This is one of the realizations for me, we end up with a bunch of extra imports for the customizations, alongside existing extension imports. Do you think it would fit to re-export all things of the original plugin from the customized one? So that we end up switching out all `@backstage/plugin-catalog` imports in the app with this pacakge? Feeling it's worth giving it a go at least.

Reply:
Could we try it out in this PR to see what it looks like for the catalog plugin? I think that's probably the worst possible case too, most plugins just export one or a few extensions.